frontend/src/modules/search/components/SearchBox.tsx

```diff
@@ -162,25 +162,25 @@ export class SearchBoxBase extends React.Component<InternalProps, State> {
     }
 
     getInitialStatuses(): Record<string, boolean> {
-        const statuses = {};
+        const statuses: Record<string, boolean> = {};
```

Comment:
I'm not happy with how typing works here, but it also seemed very involved.

Basically, I think we need to get the flag lists to be some sort of enums to get better typing here, but I also didn't find any easy wins here.